src/drivers/net/bnxt/bnxt.c

```diff
@@ -82,16 +82,17 @@ static struct pci_device_id bnxt_nics[] = {
 	PCI_ROM( 0x14e4, 0x1807, "14e4-1807", "14e4-1807", BNXT_FLAG_PCI_VF ),
 	PCI_ROM( 0x14e4, 0x1808, "14e4-1808", "14e4-1808", BNXT_FLAG_PCI_VF ),
 	PCI_ROM( 0x14e4, 0x1809, "14e4-1809", "14e4-1809", BNXT_FLAG_PCI_VF ),
+	PCI_ROM( 0x14e4, 0x1760, "14e4-1760", "14e4-1760", 0 ),
```

Comment:
The `14e4-17xx` should be **before** `14e4-18xx`.

Reply:
Ack.  I have moved this new entry below the last 14e4-175x entry.  I will reorder the other out of order entries in a different commit